carball/json_parser/actor/dropshot.py

```diff
@@ -21,22 +21,22 @@ def update(self, actor: dict, frame_number: int, time: float, delta: float) -> N
         if tile_id not in self.parser.dropshot['tile_states']:
             self.parser.dropshot['tile_states'][tile_id] = 0
 
-        damage_state = actor['TAGame.BreakOutActor_Platform_TA:DamageState']['damage_state']
+        damage_state = actor['TAGame.BreakOutActor_Platform_TA:DamageState']
 
         # unknown1: 0 - undamaged, 1 - damaged, 2 - destroyed
-        state = damage_state['unknown1']
+        state = damage_state[0]
```

Comment:
This has also been fixed in the latest version. Thanks! 😄 

https://docs.rs/boxcars/0.8.1/boxcars/attributes/struct.DamageState.html